concepts/05 UI Components/Menu/00 Getting Started with Menu/00 Getting Started with Menu.md

```diff
@@ -0,0 +1,9 @@
+#include tutorials-intro-installationnote
+
+This tutorial shows how to add a Menu component to your application and configure its core features.
```

Comment:
>  This tutorial shows how to add a Menu component to **your application** and configure **its** core features.

Please note that the pronoun "its" points at "your application" rather than "a Menu component". I suspect that it's not what you originally meant.

Reply:
You see, "the Menu" and "the component" phrases are not interchangeable, because you didn't say before that Menu is a component. It's not obvious. I would have rather said the following:

> This tutorial shows how to add the Menu component to your application and configure the component's core features.